.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    env: {
+        es2021: true,
+        node: true
+    },
+    root: true,
+    extends: ["prettier"],
+    parser: "@typescript-eslint/parser",
+    plugins: ["@typescript-eslint"],
+    parserOptions: {
+        ecmaVersion: "latest",
+        sourceType: "module",
+        project: true,
+        tsconfigRootDir: __dirname
+    },
+    rules: {
+        "no-restricted-imports": ["error", "fs/promises"],
+        "func-style": ["error", "declaration", { "allowArrowFunctions": true }],
+        "max-params": ["error", 1]
```

Comment:
This isn't _ideal_, but it should help remind folks to use parameter objects.

Reply:
Once consequence was that I needed to refactor a couple of functions in the codebase to align with this, but nothing major